celerywyrm/settings.py

```diff
@@ -8,9 +8,13 @@
 REDIS_BROKER_HOST = env("REDIS_BROKER_HOST", "redis_broker")
 REDIS_BROKER_PORT = env("REDIS_BROKER_PORT", 6379)
 REDIS_BROKER_DB_INDEX = env("REDIS_BROKER_DB_INDEX", 0)
+REDIS_BROKER_URL = env(
+    "REDIS_BROKER_URL",
+    f"redis://:{REDIS_BROKER_PASSWORD}@{REDIS_BROKER_HOST}:{REDIS_BROKER_PORT}/{REDIS_BROKER_DB_INDEX}",
+)
 
-CELERY_BROKER_URL = f"redis://:{REDIS_BROKER_PASSWORD}@{REDIS_BROKER_HOST}:{REDIS_BROKER_PORT}/{REDIS_BROKER_DB_INDEX}"
-CELERY_RESULT_BACKEND = f"redis://:{REDIS_BROKER_PASSWORD}@{REDIS_BROKER_HOST}:{REDIS_BROKER_PORT}/{REDIS_BROKER_DB_INDEX}"
+CELERY_BROKER_URL = REDIS_BROKER_URL.replace("unix:", "redis+socket:")
```

Comment:
I think it would be clearer to re-build the url than the use replace here, if possible

Reply:
This is because redis-py (used everywhere else) requires `unix://` for socket urls, but celery requires `redis+socket://` for socket urls